README.md

```diff
@@ -0,0 +1,220 @@
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [awesomebooks] <a name="about-project"></a>
+
+>
+
+**[awesomebooks]** This is awesome books website, where you register a book with author's name. It also allows you to perform CRUD operations on it.
+
+## 🛠 Built With <a name="built-with">HTML, CSS and javascript</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech stack
+> I have used vanilla js and luxon.js to build this project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://css.org/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Interactivity</summary>
+  <ul>
+    <li><a href="https://js.com/">vanilla js</a></li>
+  </ul>
+</details>
+
+<details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[allows_book_registration]**
+- **[performs_CRUD]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://ibranista.github.io/awesome-books-ibranista.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- open index.html and open it inside your favorite browser.
+
+### Setup
+
+- Create a local directory that you want to clone the repository.
+
+- Open the git-bash in the current created directory.
+
+- Clone this repository
+
+- Open git-bash inside the cloned repository
+
+- run npm install
+
+- after installation has finished you can open index.html
+
+- That's it enjoy :)
+
+### Install
+
+Install this project with:
+
+install the dependancies using node package manager also known as npm.
+
+### Usage
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Ibrahim**
+
+- GitHub: [@githubhandle](https://github.com/ibranista)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
- [ ] Please consider adding 1 - 3 future features that would fit great on this project. Having a professional README will look more attractive not only to other developers that want to collaborate on your project but for future recruiters! Please work on this. 🤓 🛠️

Reply:
- [ ] It will be awesome to add the future features as suggested by the previous reviewer, it may be any thing you wish to add to improve your project in the feature. Maybe add a `search` button, `arrange the books in ascending order `.... anything you can think of. ℹ️